extensions/wikia/MercuryApi/handlers/MercuryApiMainPageHandler.class.php

```diff
@@ -7,7 +7,11 @@ class MercuryApiMainPageHandler {
 	public static function getMainPageData( MercuryApi $mercuryApiModel ) {
 		$mainPageData = [ ];
 		$curatedContent = self::getCuratedContentData( $mercuryApiModel, null );
-		$trendingArticles = $mercuryApiModel->getTrendingArticlesData();
+		$trendingArticles = $mercuryApiModel->getTrendingPagesData(
+			10,
+			null,
+			false
```

Comment:
it's not really obvious what these three values mean

Reply:
![screenshot from 2018-11-02 13-38-44](https://user-images.githubusercontent.com/7030884/47915931-b8216800-dea4-11e8-8187-5067c99049ee.png)

![screenshot from 2018-11-02 13-39-38](https://user-images.githubusercontent.com/7030884/47915960-cc656500-dea4-11e8-99b7-158e49668406.png)